steps/appcenter-ipa-upload/0.0.1/step.yml

```diff
@@ -0,0 +1,56 @@
+title: AppCenter ipa upload
+summary: |
+  Upload ipa to Microsoft AppCenter
+description: |
+  This step is upload iOS ipa archive file to AppCenter.
+  Using AppCenter CLI.
+website: https://github.com/kanari3/bitrise-step-appcenter-ipa-upload
+source_code_url: https://github.com/kanari3/bitrise-step-appcenter-ipa-upload
+support_url: https://github.com/kanari3/bitrise-step-appcenter-ipa-upload/issues
+published_at: 2019-06-24T04:25:45.449305+09:00
+source:
+  git: https://github.com/kanari3/bitrise-step-appcenter-ipa-upload.git
+  commit: 749f916d65d674c3dd2bd714bfa325ac49db2d4c
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+project_type_tags:
+- ios
+type_tags:
+- deploy
+toolkit:
+  bash:
+    entry_file: step.sh
+deps:
+  brew:
+  - name: git
+  - name: wget
+  apt_get:
+  - name: git
+  - name: wget
+is_requires_admin_user: true
+is_always_run: false
+is_skippable: false
+run_if: ""
+inputs:
+- app_id: $APP_CENTER_APP_ID
+  opts:
+    description: |
+      AppCenter Application Identifier is a combination of your Organization/Owner Name
+      and an Application Name.
+      For example: "MyOrganization/MyApplication"
+    is_required: true
+    summary: AppCenter application identifier (Owner + App Name).
+    title: Application Identifier
+- api_token: $APP_CENTER_ACCESS_TOKEN
```

Comment:
please mark this input as sensitive, you can find further info about it in our devcenter or step development guideline